docs/i18n/espanol/codebase-best-practices.md

```diff
@@ -105,21 +105,21 @@ export const reducer = (
 Dentro de un componente, importa las acciones y los selectores necesarios.
 
 ```tsx
-// Add type definition
+// Añada el tipo de definición
 interface MyComponentProps {
   actionFunction: typeof actionFunction;
 }
-// Connect to Redux store
+// Conecta a Redux store
 const mapDispatchToProps = {
   actionFunction
 };
-// Example React Component connected to store
+// Ejemplo del componente de React conectado a store
 const MyComponent = ({ actionFunction }: MyComponentProps): JSX.Element => {
   const handleClick = () => {
-    // Dispatch function
+    // Envío de la función
     actionFunction();
   };
-  return <button onClick={handleClick}>freeCodeCamp is awesome!</button>;
+  Devuelve <button onClick={handleClick}>freeCodeCamp is awesome!</button>;
```

Comment:
This should not happen

Reply:
Good catch, Shaun.  This shouldn't even be possible.  @nhcarrigan any clue what went wrong here?

Reply:
Thinking about it, I would have thought the comments would not be translatable either, since they're all in a triple backticked code block.

Reply:
Because the docs use code blocks within code blocks, Crowdin doesn't parse them in the same way and we can't automatically hide them.